invenio_accounts/ext.py

```diff
@@ -78,6 +78,7 @@ def init_config(self, app):
             app.config.setdefault("ACCOUNTS_USE_CELERY", False)
 
         app.config.setdefault('ACCOUNTS', True)
+        app.config.setdefault('DB_USER_MODEL', User)
```

Comment:
or an entry point as mentioned in https://github.com/inveniosoftware/invenio-db/pull/21#discussion_r47119393